helm/stacks-blockchain/values.yaml

```diff
@@ -0,0 +1,360 @@
+  #   rpc_bind = "0.0.0.0:20443"
+  #   # Port should match node.p2pPort
+  #   p2p_bind = "0.0.0.0:20444"
+  #   seed = "replace-with-your-private-key-if-mining"
+  #   miner = false
+
+  #   [burnchain]
+  #   chain = "bitcoin"
+  #   mode = "krypton"
+  #   peer_host = "bitcoind.blockstack.org"
+  #   #process_exit_at_block_height = 5340
+  #   #burnchain_op_tx_fee = 5500
+  #   #commit_anchor_block_within = 10000
+  #   rpc_port = 18443
+  #   peer_port = 18444
+
+  #   [[mstx_balance]]
+  #   address = "STB44HYPYAT2BB2QE513NSP81HTMYWBJP02HPGK6"
+  #   amount = 10000000000000000
+  #   [[mstx_balance]]
+  #   address = "ST11NJTTKGVT6D1HY4NJRVQWMQM7TVAR091EJ8P2Y"
+  #   amount = 10000000000000000
+  #   [[mstx_balance]]
+  #   address = "ST1HB1T8WRNBYB0Y3T7WXZS38NKKPTBR3EG9EPJKR"
+  #   amount = 10000000000000000
+  #   [[mstx_balance]]
+  #   address = "STRYYQQ9M8KAF4NS7WNZQYY59X93XEKR31JP64CP"
+  #   amount = 10000000000000000
+
+  ## Annotations to be added to the Configmap
+  ##
+  annotations: {}
+
+################################
+####### Service Settings #######
+################################
+service:
+  ## The service type to create
+  ## If creating a public node for others to connect to, use Loadbalancer to ensure the advertised IP is reachable
+  ##
+  type: ClusterIP
+
+  ## Set external traffic policy to "Local" to preserve source IP on providers supporting it.
+  ## This field is only used when service.type is set to "NodePort" or "LoadBalancer".
+  ## Setting to "Local" can help the Stacks network if using service type NodePort or LoadBalancer.
+  ## Ref: https://kubernetes.io/docs/tutorials/services/source-ip/#source-ip-for-services-with-typeloadbalancer
+  ##
+  externalTrafficPolicy: Local
+
+  ## Will use node.rpcPort if omitted
+  ##
+  rpcPort:
+
+  ## Will use node.p2pPort if omitted
+  ##
+  p2pPort:
+
+  ## Annotations to be added to the Service
+  ##
+  annotations: {}
+
+################################
+### Service Account Settings ###
+################################
+serviceAccount:
+  
+  ## Specifies whether a ServiceAccount should be created
+  ## If true, the stacks-blockchain node will be ran using this ServiceAccount
+  ##
+  create: true
+  
+  ## Annotations to add to the service account
+  ##
+  annotations: {}
+  
+  ## The name of the service account to use.
+  ## If not set and create is true, a name is generated using the fullname template.
+  ##
+  name: ""
+
+################################
+######## RBAC Settings #########
+################################
+rbac:
+  ## Specifies whether a Role should be created
+  ##
+  create: false
+
+  ## Rules to add to the Role resource bound to the ServiceAccount
+  ##
+  rules: []
+    # - apiGroups:
+    #     - ""
+    #   verbs:
+    #     - get
+    #     - list
+    #     - watch
+    #   resources:
+    #     - pods
+
+################################
+####### Metrics Settings #######
+################################
+metrics:
```

Comment:
I would be more clear here that these metrics are for prometheus (with a link to the prom chart maybe, or prom main site), rather than assume a user knows this is for prometheus

Reply:
Added more info and a reference to prometheus docs